src/config/mod.rs

```diff
@@ -169,7 +169,8 @@ create_config! {
             "Enables unstable features. Only available on nightly channel";
     disable_all_formatting: bool, false, true, "Don't reformat anything";
     skip_children: bool, false, false, "Don't reformat out of line modules";
-    hide_parse_errors: bool, false, false, "Hide errors from the parser";
+    hide_parse_errors: bool, false, false, "(deprecated: use show_parse_errors instead)";
+    show_parse_errors: bool, true, true, "Show errors from the parser";
```

Comment:
Renaming `hide_parse_errors` -> `show_parse_errors` should not change the `stability` of the option. Please mark this option as unstable.
```suggestion
    show_parse_errors: bool, true, false, "Show errors from the parser";
```